test/rubygems/test_gem_version.rb

```diff
@@ -78,12 +78,17 @@ def test_initialize
   end
 
   def test_initialize_bad
-    %W[
+    bad = %W[
```

Comment:
Could the local variable array be called `bad_specifiers` or `bad_version_specifiers`?

(This would make the `each` block parameter `bad` read a lot clearer.)